client/reader/stream/x-post.jsx

```diff
@@ -80,9 +80,10 @@ class CrossPost extends PureComponent {
 
 	handleClick = ( event ) => {
 		if ( event.button > 0 || event.metaKey || event.controlKey || event.shiftKey || event.altKey ) {
-			return;
+			window.open( this.props.post.URL, '_blank', 'noopener noreferrer' );
```

Comment:
we can't just use window.open. That breaks everything else that could be in the context menu and doesn't respect the choice the user makes. The point of this bit is to allow users to right-click on a link or command / shift / option click on a link and do the normal browser behavior.